portfolio/src/main/webapp/style.css

```diff
@@ -1,12 +1,24 @@
 /* ----ALL PAGES---- */
 html{
-    background-color:#F7F7F7;
+    background-color:#f7f7f7;
     color:#3F3F44;
     font-family: 'Noto Sans JP', sans-serif;
 }
 a{
     text-decoration:none;
 }
+button{
+    background-color:#CCEABB;
```

Comment:
you changed the style background color above to use all lower case which is good! The google style guide for CSS recommends that. Please convert all color codes to lower case here and throughout.

Reply:
Ok! All colors should be in lower case now.